src/main/scala/exercises/04_Options.scala

```diff
@@ -34,16 +34,27 @@ object Options {
    * - does not exist: 					"not existing"
    */
   def roomState(rooms: Map[Int, Option[String]], room: Int): String = {
-    error("Fix me")
+    rooms get room match {
```

Comment:
I believe get is safer since it returns the `Option` type. Consider the following example:

```scala
val myMap = Map(2016->"Hello, World!")

// Lookup something that exists
(myMap get 2016) == Some("Hello, World!")

// Lookup something that does NOT exist
(myMap get 2017) == None
 
// Access something that exists
myMap(2016) == "Hello, World!"

// Access something that doesn't exist....
myMap(2017) == "Hello, World!"

// That last line of code will throw an error specifically with the myMap(<some_non_existent_key>) call:
// java.util.NoSuchElementException: key not found: 2017
//  at scala.collection.MapLike$class.default(MapLike.scala:228)
//  at scala.collection.AbstractMap.default(Map.scala:59)
//  at scala.collection.MapLike$class.apply(MapLike.scala:141)
//  at scala.collection.AbstractMap.apply(Map.scala:59)
 // ... 32 elided
```

> NB that the return types are different, but it's more about the safety of the access.